control-plane/pkg/reconciler/broker/broker.go

```diff
@@ -220,7 +227,24 @@ func (r *Reconciler) finalizeKind(ctx context.Context, broker *eventing.Broker)
 		// eventually be seen by the dispatcher pod and resources will be deleted accordingly.
 	}
 
-	topic, err := r.deleteTopic(broker)
+	config, err := r.resolveBrokerConfig(logger, broker)
+	var bootstrapServers []string
+	if err != nil {
+		// An error occurred, try to use default bootstrap servers.
+		r.bootstrapServersLock.RLock()
+		bootstrapServers = r.bootstrapServers
+		r.bootstrapServersLock.RUnlock()
+
+		logger.Error(
```

Comment:
I don't think any error logging is needed here, it's a pretty normal use case not overriding the bootstrap servers right?

Reply:
Yeah, make sense